task/net/src/server.rs

```diff
@@ -56,7 +56,7 @@ impl<'a> ServerImpl<'a> {
         self.socket_handles
             .get(index)
             .cloned()
-            .ok_or(RequestError::Fail(ClientError::BadMessage))
+            .ok_or(RequestError::Fail(ClientError::BadMessageContents))
```

Comment:
Is this change intentionally in this PR?

Reply:
It is! It's a side effect of the idol-runtime bump. This was the only place in the code that used that enum variant.